src/cobalt.rs

```diff
@@ -42,6 +42,13 @@ pub fn build(config: &Config) -> Result<()> {
     let dest = PathBuf::from(&config.dest).join("");
     let dest = dest.as_path();
 
+    // try to create the supplied dest dir if it currently does not exist
+    if dest.is_dir() == false {
+        try!(fs::create_dir(dest));
+        info!("Created dest dir: {:?}", dest);
+    }
```

Comment:
I'm pretty sure you can remove this again. With your new test it should be easy to verify. Sorry for the confusion. :)